src/Text/Pandoc/Writers/JATS.hs

```diff
@@ -405,8 +408,11 @@ inlineToJATS opts (Link (ident,_,kvs) txt ('#':src, _)) = do
              [("alt", stringify txt) | not (null txt)] ++
              [("rid", src)] ++
              [(k,v) | (k,v) <- kvs, k `elem` ["ref-type", "specific-use"]]
-  contents <- inlinesToJATS opts txt
```

Comment:
This change seems unrelated to the main change of this commit.  Please split into two separate commits, and ideally two separate PRs, one for each logical change.